front-end/src/atoms/RefreshButton.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import "./Button.css";
+import "./RefreshButton.css";
+
+const RefreshingButton = ({ child, onClick }) => (
```

Comment:
UI looks good! Approving but can this be changed to be the same as the component name pls